### DISCUSSION TITLE
A just work vdd program for sunshine

### DISCUSSION THREAD

**Discussion by VergilGao** [LizardByte/community#389] [Sunshine]
https://github.com/VergilGao/vddswitcher

I'm sorry I don't have the ability to write an English README

Key features:

- Create a virtual display when you start streaming

- Automatically sets the resolution of the virtual display based on the resolution set by the Moonlight client (if supported)

- Turn off the virtual display when you end streaming

**Comment by ReenigneArcher** [comment]
@FrogTheFrog is this useful for the stuff you are working on?

**Reply by FrogTheFrog** [reply]
Nope (in a sense that it does not help my feature).

It's a completely novel approach though. Theoretically it could be another feature, where we create a display device and just switch to it.

[a comment by an automated account is not shown]
